state/execution.go

```diff
@@ -132,6 +134,23 @@ func (blockExec *BlockExecutor) ApplyBlock(
 	state State, blockID types.BlockID, block *types.Block,
 ) (State, int64, error) {
 
+	profile_path := os.Getenv("DAEMON_PROFILE_PATH")
```

Comment:
This is awesome! Maybe rename to `"PPROF_BLOCK_EXEC_PROFILE_PATH", but this would be extremely useful to have for more visibility.

Have found traces you produced like this extremely helpful several times

Reply:
done!